syntax = "proto3";

package iaservice;

service DriverMonitor {
  rpc AnalyzeDriver (ImageRequest) returns (AnalysisReply) {}
}

message ImageRequest {
  bytes image_data = 1;
}

message AnalysisReply {
  string status = 1;
  string emotion = 2;
  float confidence = 3;
}